env.sh

```diff
@@ -68,7 +68,9 @@ fi
 export CLANG_COMPATIBLE_FLAGS=`echo | ${STARROCKS_GCC_HOME}/bin/gcc -Wp,-v -xc++ - -fsyntax-only 2>&1 \
                 | grep -E '^\s+/' | awk '{print "-I" $1}' | tr '\n' ' '`
 
-# check java home
+if [[ -z ${JAVA_HOME} ]]; then
+    JAVA_HOME="$(dirname $(dirname $(readlink -f $(which javac))))"
```

Comment:
Better to print a log to indicate that this JAVA_HOME is inferred.

Reply:
need export JAVA_HOME

Reply:
updated